src/docs/reference/modules/Roles/README.md

```diff
@@ -4,7 +4,7 @@ Enabling the `OrchardCore.Roles` module will allow you to manage the user roles.
 
 ## Predefined Roles
 
-Orchard Core come up with the following predefined roles:
+Orchard Core come up with the following predefined permission stereotypes:
```

Comment:
If you set up the site with the Blog recipe, you'll get all these roles though, and only `Anonymous` is created (or rather, updated) from the recipe.

Reply:
Yes, you are right. I found this: https://github.com/OrchardCMS/OrchardCore/blob/339d74b251b53393dd510f8ad3027aa17d78500d/src/OrchardCore.Modules/OrchardCore.Roles/Services/RoleUpdater.cs#L50

So if a feature has permission stereotypes, the Roles module creates (or updates) roles out of them if I'm correct. But in this sense roles are not predefined, rather they are created on-demand out of predefined permission stereotypes.

Reply:
Ah I see, nice catch.